docs/release-notes/release-notes-0.19.0.md

```diff
@@ -90,6 +90,9 @@
 * The `lncli wallet fundpsbt` command now has a [`--max_fee_ratio` argument to
   specify the max fees to output amounts ratio.](https://github.com/lightningnetwork/lnd/pull/8600)
 
+* The `lncli addinvoice --blind` command now has the option to include a chained
+  channels incoming list [`--blinded_path_incoming_channel_list](https://github.com/lightningnetwork/lnd/pull/9127)
```

Comment:
Nit: Can you wrap to 80 lines (I know some other entries are also not wrapped but I try to wrap them here as well)